### DISCUSSION TITLE
An issue of deploying multiple secondary virtual directories under the same domain name

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/Uno.Wasm.Bootstrap#607] [Q&A] [answered]
When deploying wasm, if multiple second-level virtual directories exist under the same domain name, the files cached in IndexedDB will affect each other because the browser manages the storage on a domain name basis!

@jeromelaban Thanks!

**Comment by jeromelaban** [comment, marked as answer]
Interesting issue. There does not seem to be any work around [this in emscripten](https://emscripten.org/docs/api_reference/Filesystem-API.html).
